site/docs-md/cordova/using-cordova-plugins.md

```diff
@@ -0,0 +1,51 @@
+# Using Cordova Plugins and Ionic Native
+
+When developing an app that uses Capacitor, it's possible to use both Cordova and Ionic Native plugins.
+
+## Installing Cordova Plugins
+
+Simply install your plugin of choice, sync your project, finish any required native project configuration, and you're ready to go:
+
+```bash
+npm install cordova-plugin-name
+npx cap sync
+```
+
+## Updating Cordova Plugins
+
+Similar to the installation steps. Capacitor will detect the new version of the plugin and install it.
+
+```bash
+npm install cordova-plugin-name
+npx cap update
+```
+
+## Installing Ionic Native Plugins
+[Ionic Native](https://ionicframework.com/docs/native) provides TypeScript wrappers and a consistent API and naming convention for easier development with Cordova plugins. It's supported in Capacitor, so whenever you find an Ionic Native wrapper you'd like to use, install the JavaScript code, install the corresponding Cordova plugin, then sync your project:
+
+```bash
+npm install @ionic-native/javascript-package-name
+cordova plugin add cordova-plugin-name
```

Comment:
better tell them to use `npm install cordova-plugin-name` as they might not have a cordova project

Reply:
Makes sense, now that I understand that package.json, **not** config.xml is the file that Cap reads. Had those mixed up.